### ISSUE TITLE
Strongly Typed Content Item Converter does not support mapping name from an Option

### ISSUE THREAD

**Issue by aweigold** [Kentico/delivery-sdk-java#23]
If an Option is mapped to a field, the name from the option should be used.

**Comment by cunhazera** [comment]
@aweigold can you give some example of how to reproduce the bug?

**Comment by aweigold** [comment]
This came from me working with the API and finding that Options (and Assets for that matter) don't provide a good way to simply map to a string in a strongly typed model.  I could see it being useful for someone to just have a field set based on an option being set, without exposing the user to having to use the Option class.  For example, if the option is a radio button, it would make sense to just set a String value of that Option's selection.  On the Asset side, this would be mapping the URL or description String directly to a field.

Given that I don't believe the .Net SDK supports this kind of smart mapping, this may not be a priority, but I wanted to capture the issue as a nice to have.

**Comment by petrsvihlik** [comment]
You're right, such functionality is not present in the [.NET SDK](https://github.com/Kentico/delivery-sdk-net).

But let's say you wanted to render radio buttons - wouldn't you use `Option.name` for labels and `Option.codename` for ids?

`<input type="radio" name="groupname" value="Option.codename" />Option.name<br>`

**Comment by aweigold** [comment]
@petrsvihlik I think the usage of Option is fine, and probably could change this from a 'bug' to 'enhancement'.  It all goes back to if we want to require a consumer to use our object model.  I would beg the question "What's the difference between allowing a StronglyTyped ContentItem versus a StronglyTyped Option or Asset?"

In the end, I don't really care that much, we could close this as well. 😄

**Comment by petrsvihlik** [comment]
right :) let's change it to enhancement...that should be enough for now :)
